benchmarks/asv.conf.json

```
@@ -16,5 +16,6 @@
     "env_dir": "env",
     "results_dir": "results",
     "html_dir": "html",
-    "build_cache_size": 8
+    "build_cache_size": 8,
+    "default_benchmark_timeout": 1200
```

```suggestion
    "default_benchmark_timeout": 1200,
```
You can put a comma at the end of the list... Python is ok with that. And it means you don't have to change that line when you add the inevitable next key/value pair to the dict. Same thing works for lists and tuples.